app/app_test.go

```diff
@@ -81,8 +82,9 @@ func (at *appTest) reset() {
 	require.True(at.t, resabci.IsOK(), resabci)
 }
 
-func getBalance(key basecoin.Actor, state state.KVStore) (coin.Coins, error) {
-	acct, err := coin.NewAccountant("").GetAccount(state, key)
+func getBalance(key basecoin.Actor, store state.KVStore) (coin.Coins, error) {
+	cspace := stack.PrefixedStore(coin.NameCoin, store)
+	acct, err := coin.GetAccount(cspace, key)
```

Comment:
**Important**: it seems like fundamentally the approach that is being used here may be fundamentally insecure. Question to answer: if I rig up a custom plugin to attempt to use a state space other than its designated prefixed space, how is basecoin going to reject its request? It seems like we should enforce use of the `PrefixedStore` throughout the application by feeding the plugin a `PrefixedStore` instead of a regular store and then hoping that the plugin is going to add the required prefix